styles/pronouns.cc/catppuccin.user.css

```diff
@@ -0,0 +1,235 @@
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root[data-bs-theme="light"] {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    .bg-light when (@lookup = latte) {
+      --bs-bg-opacity: 1;
+      background-color: rgba(
+        var(--bs-darkk-rgb),
+        var(--bs-bg-opacity)
+      ) !important;
+    }
+
+    .bg-dark when not (@lookup = latte) {
+      --bs-bg-opacity: 1;
+      background-color: rgba(
+        var(--bs-lightt-rgb),
+        var(--bs-bg-opacity)
+      ) !important;
+    }
+    .text-muted {
+      --bs-text-muted: @subtext0;
+      color: @subtext0 !important;
+    }
+    .form-floating input::placeholder {
+      color: transparent !important;
+    }
+    .form-control:focus {
+      border-color: @accent-color;
+    }
+    --bs-border-color: @surface1;
+    a {
+      --bs-link-color-rgb: #rgbify(@blue) [] !important;
+    }
+    .modal {
+      --bs-modal-bg: @crust;
+      --bs-modal-border-color: var(--bs-border-color-translucent);
+      --bs-modal-header-border-color: var(--bs-border-color);
+      --bs-modal-footer-border-color: var(--bs-border-color);
+    }
+    .toast {
+      --bs-toast-color: @text;
+      background-color: darken(@surface0, 5%);
+    }
+    .toast-header {
+      color: @text;
+      background-color: @surface0;
+    }
+    .toast-body {
+      color: @text;
+    }
+    --bs-badge-color: @text;
+    --bs-link-color-rgb: #rgbify(@blue) [];
+    --bs-body-color: @text;
```

Comment:
```suggestion
    --bs-body-color: @text;
    --bs-body-color-rgb: #rgbify(@text) [];
```

Fixes this input label:

![CleanShot 2024-10-11 at 13 37 51](https://github.com/user-attachments/assets/917099ef-d4e3-492b-b41a-f79c2f7a2e35)